cabal.project

```diff
@@ -76,10 +76,6 @@ if os(windows)
 package bitvec
   flags: -simd
 
--- See the note on nix/agda-packages.nix for why this is here.
--- (NOTE this will change to ieee754 in newer versions of nixpkgs).
-extra-packages: ieee, filemanip
```

Comment:
Obsolete: `adga-pacakges.nix` uses `cabalProjectLocal` to add `ieee` and `filemanip`